modules/RTC/TraceablePeerConnection.js

```diff
@@ -406,48 +406,49 @@ function extractSSRCMap(desc) {
     if (typeof desc !== 'object' || desc === null ||
         typeof desc.sdp !== 'string') {
         logger.warn('An empty description was passed as an argument.');
-        return desc;
+        return;
     }
 
-    var ssrcList = {};
-    var ssrcGroups = {};
-    var session = transform.parse(desc.sdp);
-    if (!Array.isArray(session.media))
-    {
+    const ssrcList = { };
```

Comment:
use a Map? Also ssrcMap sounds like a more appropriate name, since it's not a list